src/main/java/gavinchatbot/task/ToDos.java

```diff
@@ -0,0 +1,37 @@
+package gavinchatbot.task;
+
+/**
+ * Represents a task that is a to-do.
+ * Inherits from the Task class.
+ */
+public class ToDos extends Task {
```

Comment:
Perhaps this class would be better named as `ToDo`

Reply:
Thank you Sparsh, I will take note!